tests/modules/annotations/resources/test_matching_set.py

```diff
@@ -324,3 +325,109 @@ def test_annotation_matching_set_macros(
     assert len(replaced['filter']['C']['match']['should']) == len(vps)
     for vp in vps:
         assert {'term': {'viewpoint': vp}} in replaced['filter']['C']['match']['should']
+
+
+@pytest.mark.skipif(
+    module_unavailable('asset_groups'),
+    reason='AssetGroups module disabled',
+)
+@pytest.mark.skipif(
+    extension_unavailable('elasticsearch'),
+    reason='Elasticsearch extension or module disabled',
+)
+def test_search_with_elasticsearch(
+    flask_app_client,
+    researcher_1,
+    admin_user,
+    request,
+    test_root,
+):
+    # pylint: disable=invalid-name
+    from app.extensions import elasticsearch as es
+    from app.modules.annotations.models import Annotation
+
+    uuids = enc_utils.create_encounter(flask_app_client, researcher_1, request, test_root)
+    enc_guid = uuids['encounters'][0]
+    asset_guid = uuids['assets'][0]
+
+    tx = setting_utils.get_some_taxonomy_dict(flask_app_client, admin_user)
+    assert tx
+    assert 'id' in tx
+    taxonomy_guid = tx['id']
+    locationId = 'erehwon'
```

Comment:
Is it worth doing a test with a locationId that is a GUID also?

Reply:
Fixed